### ISSUE TITLE
Need manually trigger pipeline after first time push to codecommit

### ISSUE THREAD

**Issue by mouyigang** [awslabs/aws-cdk#1310]
I found myself in this situation:

1. Create codecommit repository, codebuild project and codepipeline through cdkapp and deploy.
2. Push my code to the codecommit repository hoping it will trigger the codepipeline.
3. The codepipeline never get triggered.

I have to manually retry on the source stage once.

I tried to manually reproduce this issue, but codepipeline doesn't allow you to create new pipeline without a branch already exist in codecommit repo, though cdk allow a fallback to "master".

I wonder if this is the case, or I'm using it wrong.

Here is my code:

```js
const sourcerepo = new codecommit.Repository(this, 'myrepo' ,{
            repositoryName: 'myrepo'
 });

const codebuildproject = new codebuild.Project(this,  'mybuild', {
        projectName: 'mybuild',
        source: new codebuild.CodeCommitSource({
            repository: sourcerepo
       })
})
const pipeline = new codepipeline.Pipeline(this, 'mypipeline', {
        pipelineName: 'mypipeline'
});
const sourceStage = pipeline.addStage('Source');
sourcerepo.addToPipeline(sourceStage,  'CodeCommit');
const buildStage = pipeline.addStage('Build');
new codebuild.PipelineBuildAction(this, 'mybuildstage', {
        stage: buildStage,
        project: codebuildproject
});
```

**Comment by mouyigang** [comment]
Do you need more information regarding to this question?

**Comment by skinny85** [comment]
Nope, it was just the weekend here @mouyigang . I'll take a look at this today.

**Comment by skinny85** [comment]
You were right @mouyigang , I've managed to reproduce your problem. I've submitted a fix for it in PR: #1320
